Bots/AuthServer.js

```diff
@@ -1,59 +1,52 @@
-"use strict";
-
-var cookieParser = require('cookie-parser');
-var session = require('express-session');
-var bodyParser = require('body-parser');
-var express = require('express');
-var app = express();
-var router = express.Router();
-var passport = require('passport');
-var TwitterStrategy = require('passport-twitter').Strategy;
-
-var sessinCredential = require('common/makeCredential')('session');
-var twitterCredential = require('common/makeCredential')('twitter');
-var AuthBot = require('models/AuthBot');
-var log = require('common/log');
-
-function AuthServer(){
-
-  passport.serializeUser(function(user, done) {
+import express from 'express';
+import cookieParser from 'cookie-parser';
+import session from 'express-session';
+import bodyParser from 'body-parser';
+import passport from 'passport';
+import { Strategy } from 'passport-twitter';
+import AuthBot from 'models/AuthBot';
+import log from 'common/log';
+
+const app = express();
+const router = express.Router();
+const sessinCredential = process.env.KINDLIZED_SESSION;
+
+function AuthServer() {
+  passport.serializeUser((user, done)=> {
     done(null, user);
   });
-
-  passport.deserializeUser(function(obj, done) {
+  passport.deserializeUser((obj, done)=> {
     done(null, obj);
   });
 
-  passport.use(new TwitterStrategy({
-      consumerKey: twitterCredential.consumerKey,
-      consumerSecret: twitterCredential.consumerSecret,
-      callbackURL: "http://localhost:3000/twitter/callback"
-    },
-    function(token, tokenSecret, profile, done) {
-      log.info(profile);
-      AuthBot.findOne({ screen_name: profile.username}, function(err, bot){
-        if(err){
-          return log.info(err);
-        }
-        if(bot){
-          return log.info("This bot is already existed.");
+  passport.use(new Strategy({
+    consumerKey: process.env.KINDLIZED_TW_CONSUMER_KEY,
+    consumerSecret: process.env.KINDLIZED_TW_CONSUMER_SECRET,
+    callbackURL: 'http://localhost:3000/twitter/callback',
+  },
+  function authCallback(token, tokenSecret, profile, done) {
+    AuthBot.findOne({ screen_name: profile.username}, (err, bot)=> {
+      if (err) {
+        return log.info(err);
+      }
+      if (bot) {
+        return log.info('This bot is already existed.');
+      }
+      const newAuthBot = new AuthBot({
+        screen_name: profile.username,
+        user: profile._json,
+        accessToken: token,
+        accessTokenSecret: tokenSecret,
+      });
+      newAuthBot.save((saveError)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).